_posts/2021-01-15-miguel-callbacks-yes-or-not.md

```diff
@@ -0,0 +1,43 @@
+---
+layout: post
+title: Can we teach Node Js without callbacks?
+image: 2021-01-15-miguel-callbacks-yes-or-not.png
+date: 2021-01-15
+type: post
+published: true
+status: publish
+category: web
+tags: [web]
+author:
+  login: miguelgamboa
+  email: 
+  display_name: Miguel Gamboa
+  image: miguel.jpg
+  role: Software Developer
+---
+
+After another semester teaching Web Development the following question arises to
+my mind: _can we teach Node Js without callbacks_?
+Considering that we cannot avoid Promises and async/await, so should we
+continually introduce callbacks in the first place? Notice that:
+1. Most part of Node API already supports Promises, e.g. file system module `fs`
+2. Starting lecturing with a simple use case of an HTTP client it is
```

Comment:
Odd statement. No easy suggestion. Probably needs some restructuring.

Reply:
New proposal: _It is increasingly difficult to find an HTTP client Node module that provides callback-based API. For instance, one of those most used modules...._